packages/ui/theme-builder/app/page.tsx

```diff
@@ -1,3 +1,31 @@
+import { SignedIn, SignedOut, SignInButton, SignOutButton, SignUpButton } from '@clerk/nextjs';
+
+function Button({ children }: { children: React.ReactNode }) {
+  return (
+    <button className='relative isolate rounded border bg-white bg-gradient-to-b from-white to-neutral-50 px-2.5 py-1 text-sm font-medium'>
+      {children}
+    </button>
+  );
+}
+
 export default function Home() {
-  return null;
+  return (
+    <div className='relative'>
+      <SignedIn>
+        <SignOutButton>
+          <Button>Sign out</Button>
+        </SignOutButton>
+      </SignedIn>
+      <SignedOut>
+        <div className='flex flex-col gap-4'>
+          <SignInButton>
+            <Button>Sign in</Button>
+          </SignInButton>
+          <SignUpButton>
+            <Button>Sign up</Button>
+          </SignUpButton>
+        </div>
```

Comment:
What if we replaced this with links to our views? Got a little caught off guard when it redirected to the old UI!

Reply:
I'll take care of this in https://linear.app/clerk/issue/SDKI-142/theme-builder-redirect-signin-and-signup-buttons-to-newly-added